components/ca-form.js

```diff
@@ -1425,7 +1422,7 @@ define([
         schemaItemToHtmlElement(key, item) {
 
             const dataType = (item.items && item.items.type) || item.type;
-            const format = (item.items && item.items.format) || item.format || ''.toLowerCase();
+            const format = item.items && item.items.format || item.format || ''.toLowerCase();
```

Comment:
This is still different to the original:
`'A' || 'B' || ''.toLowerCase() // new version`
`"A"`
`('A' || 'B' || '').toLowerCase() // previous version`
`"a"`